allure-docker-api/app.py

```diff
@@ -89,7 +90,13 @@ def index():
 @app.route("/allure-docker-service/swagger.json", strict_slashes=False)
 def swagger_json():
     try:
-        return send_file("{}/swagger.json".format(STATIC_CONTENT), mimetype='application/json')
+        if URL_PREFIX:
```

Comment:
This seems al little hacky but I couldn't figure out a better way to dynamically update the server that the swagger ui is using to include the prefix. Without this, the swagger ui is looking making api calls to the wrong endpoints.